Ranking Criteria updates (#2345)
ppy/osu-wiki: wiki/Ranking_Criteria/osu!catch/en.md

- **Hyperdashes of any kind are disallowed.** This is to ensure a manageable step in difficulty for novice players.
- **Dashes may be used if the time between the ticks of the desired snapping is 125ms or higher.** As an example, 1/3 dashes would be allowed at 160 BPM and below, whereas 1/4 dashes would be allowed at 120 BPM and below.
- **Edge dashes must not be used** as they require extremely precise timing which cannot be expected of less-experienced players.

> - **Basic dashes can't be used more than two times between consecutive fruits.** If higher-snapped dashes are used, they must be used singularly (not in conjunction with other dashes).

Reviewer: ```suggestion
- **Basic dashes must not be used more than two times between consecutive fruits.** If higher-snapped dashes are used, they must be used singularly (not in conjunction with other dashes).
```

- **Hyperdashes cannot be used on individual drops and/or slider repititions.** The accuracy and control required is unreasonable at this level and can create a situation where the player potentially fails to read the sliderpath.
- **Hyperdashes may be used if the time between the ticks of the desired snapping is 125ms or higher.** As an example, 1/3 hyperdashes would be allowed at 160 BPM and below, whereas 1/4 hyperdashes would be allowed under 120 BPM.

> - **Basic hyperdashes can't be used more than two times between consecutive fruits.** If higher-snapped hyperdashes are used, they must be used singularly (not in conjunction with other hyperdashes or dashes).

Reviewer: ```suggestion
- **Basic hyperdashes must not be used more than two times between consecutive fruits.** If higher-snapped hyperdashes are used, they must be used singularly (not in conjunction with other hyperdashes or dashes).
```

- **Hyperdashes of different beat snap (1/2, 1/4, etc.) must not be used between consecutive fruits.**
- **Basic hyperdashes may be used in conjunction with antiflow patterns.** If used, the spacing must not exceed a distance snap of 1.2 times the trigger distance when followed by a walkable movement, or 1.1 times the trigger distance when followed by a basic dash.
- **Dashes may be used if the time between the ticks of the desired snapping is 62ms or higher.** As an example, 1/6 dashes would be allowed at 160 BPM and below, whereas 1/8 dashes would be allowed at 120 BPM and below.

> - **Basic dashes can't be used more than four times between consecutive fruits.** Higher-snapped dashes can be used up to two times between consecutive fruits, provided there isn't a direction change between them.

Reviewer: ```suggestion
- **Basic dashes must not be used more than four times between consecutive fruits.** Higher-snapped dashes can be used up to two times between consecutive fruits, provided there isn't a direction change between them.
```

- **Edge dashes must not be used** as they require extremely precise timing which cannot be expected of less-experienced players.
- **At least 125 ms must be left between objects and the start of spinners to ensure readability.** As an example, a 1/4 gap would be required at 120 BPM, whereas a 1/3 gap would be required at 160 BPM.
- **At least 250 ms must be left between objects and the end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 bpm, whereas a 1/2 gap would be required at 120 BPM.

#### Guidelines

- **All distances should be clear on whether they require the player to walk or dash.** This is to ensure that players can easily recognize patterns that require dashing.

> - **Higher-snapped hyperdashes should not be followed by antiflow patterns.** If used, the spacing must not exceed a distance snap of 1.1 times the trigger distance and the movement after the hyperdash must be walkable.

Reviewer 2: ```suggestion
- **Higher-snapped hyperdashes should not be followed by antiflow patterns.** If used, the spacing should not exceed a distance snap of 1.1 times the trigger distance and the movement after the hyperdash must be walkable.
```
I messed up in my latest proposal.

- Combos should not exceed 16 objects including slider tails and repeats. Spinners are an exception.
- Note density should follow a mostly 1/2 + 1/4 and/or 1/3 + 1/6 pattern - 1/8 patterns and higher can be used sparingly.
- 1/8 and higher hyperdashes should not be used between consecutive object pairs.

> - Edge dashes may be used with caution for a maximum of three consecutive objects, and should not be used in after hyperdashes.

Reviewer: ```suggestion
- Edge dashes may be used with caution for a maximum of three consecutive objects, and should not be used after hyperdashes.
```
